### PR TITLE
Create plot: Van-der-Waals state equation (pressure from temperature and pressure from volume)

### PR DESCRIPTION
**Pull request by Voldoned**

### CODE REVIEW

File: plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import numpy as np
+from sympy import symbols, Eq, solve
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import print_expression
+from symplyphysics.laws.thermodynamics.equations_of_state import van_der_waals_state_equation as van_der_waals_law
+
+# Convert from liters to m^3
+volume_values = np.array([0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45]) * 1E-3
```
Comment (blackyblack) [inline plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py:11]: Don't use `numpy` unless really necessary.

File: plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import numpy as np
+from sympy import symbols, Eq, solve
```
Comment (blackyblack) [inline plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py:4]: CI is failing.

Comment (Voldoned) [reply plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py:4]: After a day of searching, I still don't understand why this exception occurs.

Comment (blackyblack) [reply plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py:3]: Fixed

**Review by blackyblack — APPROVED** [review]

**Pull request merged**